selenium/exercise7/Exer.rb

```diff
@@ -0,0 +1,23 @@
+require "selenium-webdriver"
+
+Selenium::WebDriver::Chrome::Service.driver_path = "C:/Users/naika/OneDrive/Documents/Drivers/chromedriver.exe"
+
+driver = Selenium::WebDriver.for :chrome
+driver.get "https://testpages.herokuapp.com/styled/basic-html-form-test.html"
+
+wait = Selenium::WebDriver::Wait.new(:timeout => 10)
+
+dropdown_ele = wait.until { driver.find_elements(:tag_name, "select") }
+dropdown_ele[1].click
+puts "click dropdown"
+
+select_ele = wait.until { dropdown_ele[1].find_elements(:tag_name, "option") }
+# select_ele.each do |op|
+#   op.click
+#   puts "clicked "
+# end
+select_ele[1].click
```

Comment:
A better approach is to use

`dropdown = Selenium::WebDriver::Support::Select.new(dropdown_xpath)`

You should make use of the functions mentioned here: https://www.selenium.dev/selenium/docs/api/rb/Selenium/WebDriver/Support/Select.html